src/styles/app/45-local-graph.css

```
@@ -54,26 +54,141 @@
+	width: 100%;
+	height: 100%;
+}
+
 .localNoteGraphViewport {
+	--local-graph-dot: color-mix(in srgb, var(--text-secondary) 18%, transparent);
+	--local-graph-surface: color-mix(
+		in srgb,
+		var(--bg-secondary) 88%,
+		var(--bg-primary)
+	);
 	width: 100%;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)